core/domain/suggestion_jobs_one_off.py

```diff
@@ -196,7 +196,12 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(item):
-        if item.deleted:
+        # Exit early if the suggestion has been marked deleted, or if the
+        # suggestion has already set the language code property, or if the
+        # suggestion type is not queryable by language, since ndb automatically
+        # sets properties that aren't intialized to None.
+        if item.deleted or item.language_code or item.suggestion_type == (
+                suggestion_models.SUGGESTION_TYPE_EDIT_STATE_CONTENT):
             return
 
         # Getting the suggestion from the model creates the corresponding
```

Comment:
Please follow [the instructions for running jobs in production](https://github.com/oppia/oppia/wiki/Running-jobs-in-production) -- we'll need to do a test run before the PR is merged.

Reply:
I think you don't need an audit job for this one.